rtv/submission_page.py

```diff
@@ -23,10 +23,15 @@ def __init__(self, reddit, term, config, oauth, url=None, submission=None):
         super(SubmissionPage, self).__init__(reddit, term, config, oauth)
 
         self.controller = SubmissionController(self, keymap=config.keymap)
+
         if url:
-            self.content = SubmissionContent.from_url(reddit, url, term.loader)
+            self.content = SubmissionContent.from_url(
+                reddit, url, term.loader,
+                max_comment_cols=config['max_comment_cols'])
         else:
-            self.content = SubmissionContent(submission, term.loader)
+            self.content = SubmissionContent(
+                submission, term.loader,
+                max_comment_cols=config['max_comment_cols'])
```

Comment:
You missed a reference to ``SubmissionContent.from_url()`` made from ``self.refresh_content()``.